In [7]:
import pandas as pd
import numpy as np

df = pd.DataFrame()
df['Names'] = ['Greg', 'Marcia', 'Peter', 'Jan', 'Bobby', 'Cindy', 'Cousin_Oliver']
df['Age'] = [14, 12, 11, 10, 8, 6, 8]

mean = np.mean(df['Age'])
median = np.median(df['Age'])
mode = df.Age.value_counts()
stdev = np.std(df['Age'], ddof = 1)
stderr = stdev / (np.sqrt(len(df['Age'])))
variance = stdev **2

print(mean)
print(median)
print(mode)
print(variance)
print(stdev)
print(stderr)

9.857142857142858
10.0
8     2
6     1
14    1
12    1
11    1
10    1
Name: Age, dtype: int64
7.476190476190476
2.734262327610589
1.0334540197243192


**#1.** <br />
Mean: 9.857<br />
Median: 10<br />
Mode: 8<br />
Variance:7.476<br />
Standard deviation: 2.734<br /> 
Standard error: 1.033<br />

**#2.** <br />
The mean is an appropriate central tendency statistic for this sample. For one, there are no outliers that would have an outsize impact on the mean estimate. There are very few data points, so the mode is not a good indication of central tendency. Also, the distribution is  skewed, so the median is not necessarily better than the mean. I would use standard deviation as it is informative about the spread around the mean.

In [36]:
df2 = pd.DataFrame()
df2['Names'] = ['Greg', 'Marcia', 'Peter', 'Jan', 'Bobby', 'Cindy', 'Cousin_Oliver']
df2['Age'] = [14, 12, 11, 10, 8, 7, 8]

mean = np.mean(df2['Age'])
median = np.median(df2['Age'])
mode = df2.Age.value_counts()
stdev = np.std(df2['Age'], ddof = 1)
stderr = stdev / (np.sqrt(len(df2['Age'])))
variance = stdev **2

print(mean)
print(median)
print(mode)
print(variance)
print(stdev)
print(stderr)

10.0
10.0
8     2
7     1
14    1
12    1
11    1
10    1
Name: Age, dtype: int64
6.333333333333333
2.516611478423583
0.9511897312113418


**#3.**<br />
Since Cindy was the youngest, increasing her age by 1 year moved her closer to the average of the bunch. This reduced the overall differences in ages of everyone and therefore reduced all measures of variance. In addition, the mean increased as the age of one person increased. The median and mode did not change. 

In [4]:
df['Names'] = ['Greg', 'Marcia', 'Peter', 'Jan', 'Bobby', 'Cindy', 'Jessica']
df['Age'] = [14, 12, 11, 10, 8, 7, 1]
df

Names  Age
0     Greg   14
1   Marcia   12
2    Peter   11
3      Jan   10
4    Bobby    8
5    Cindy    7
6  Jessica    1

In [5]:
mean = np.mean(df['Age'])
median = np.median(df['Age'])
mode = df.Age.value_counts()
stdev = np.std(df['Age'], ddof = 1)
stderr = stdev / (np.sqrt(len(df['Age'])))
variance = stdev **2

print(mean)
print(median)
print(mode)
print(variance)
print(stdev)
print(stderr)

9.0
10.0
14    1
12    1
11    1
10    1
8     1
7     1
1     1
Name: Age, dtype: int64
17.999999999999996
4.242640687119285
1.6035674514745462


**#4**<br />
Swapping Cousin Olver (age 8) with Jessica (age 1) reduced the mean and increased the estimates of variance. This is because Cousin Oliver's age was much closer to the others than Jessica's.

**#5** <br />To answer this question you need to know more about the readership of each of the magazines: Does readership overlap across magazines? Do the four magazines together represent all Americans? What share of total American readership does each magazine serve? Without these answers, it is not possible to come up with a reliable answer.  